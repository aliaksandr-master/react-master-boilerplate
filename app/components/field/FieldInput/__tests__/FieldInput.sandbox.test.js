/* eslint-env browser, jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { wrapSandbox } from '../../../../vendor/react-sandbox/sandbox-staging';
import sandboxFieldInput from '../FieldInput.sandbox';
import Test from '../../../test/Test';



describe('rendering', () => {
  wrapSandbox(sandboxFieldInput).forEach(({ name, template, props }) => {
    it(`should render sandbox "${name}"`, () => {
      const FieldInputSandboxComponent = template();

      const component = renderer.create(
        <Test>
          <FieldInputSandboxComponent {...props} />
        </Test>
      );

      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
