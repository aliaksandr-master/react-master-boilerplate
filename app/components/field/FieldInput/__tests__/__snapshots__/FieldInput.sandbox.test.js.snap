// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render sandbox "default" 1`] = `
<div>
  <div
    className={undefined}
    style={
      Object {
        "backgroundColor": "transparent",
        "cursor": "auto",
        "display": "inline-block",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": 16,
        "height": 48,
        "lineHeight": "24px",
        "muiPrepared": true,
        "position": "relative",
        "transition": "height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "width": 256,
      }
    }
  >
    <input
      disabled={false}
      id="f1485955293000"
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyPress={[Function]}
      style={
        Object {
          "WebkitOpacity": 1,
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "appearance": "textfield",
          "backgroundColor": "rgba(0,0,0,0)",
          "border": "none",
          "color": "rgba(0, 0, 0, 0.87)",
          "cursor": "inherit",
          "font": "inherit",
          "height": "100%",
          "muiPrepared": true,
          "outline": "none",
          "padding": 0,
          "position": "relative",
          "width": "100%",
        }
      }
      type="text"
      value="123"
    />
    <div>
      <hr
        aria-hidden="true"
        style={
          Object {
            "borderBottomStyle": "solid",
            "borderBottomWidth": 1,
            "borderColor": "#e0e0e0",
            "borderLeft": "none",
            "borderRight": "none",
            "borderTop": "none",
            "bottom": 8,
            "boxSizing": "content-box",
            "margin": 0,
            "muiPrepared": true,
            "position": "absolute",
            "width": "100%",
          }
        }
      />
      <hr
        aria-hidden="true"
        style={
          Object {
            "borderBottomStyle": "solid",
            "borderBottomWidth": 2,
            "borderColor": "#00bcd4",
            "borderLeft": "none",
            "borderRight": "none",
            "borderTop": "none",
            "bottom": 8,
            "boxSizing": "content-box",
            "margin": 0,
            "muiPrepared": true,
            "position": "absolute",
            "transform": "scaleX(0)",
            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;
